### PR TITLE
Survey at account deletion

### LINKED ISSUE #1531: Ask reasons for account deletion
when clicking "delete account"
image
.. open a modal and ask for reasons:

- I don't need a scheduling link anymore
- It's too expensive
- Something is broken => show textfield: "What's not working?"
- I found a better product => show textfield: "Which app and what are they doing better?"
- Other

simple list of radio buttons should be fine:
image

### PR DESCRIPTION
**Pull request by alishaz-polymath**
## What does this PR do?

Provides a redirect to the survey after user account deletion

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code and corrected any misspellings
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by PeerRich — APPROVED** [review]
great stuff!

File: pages/settings/profile.tsx [hunk 1]
```diff
@@ -124,8 +124,11 @@ function SettingsView(props: ComponentProps<typeof Settings> & { localeProp: str
     }).catch((e) => {
       console.error(`Error Removing user: ${props.user.id}, email: ${props.user.email} :`, e);
     });
-    // signout;
-    signOut({ callbackUrl: "/auth/logout" });
+    if (process.env.BASE_URL === "https://app.cal.com/") {
+      signOut({ callbackUrl: "https://cal.com/cancellation" });
```
Comment (zomars) [inline pages/settings/profile.tsx:128]:
```suggestion
    if (process.env.NEXT_PUBLIC_BASE_URL === "https://app.cal.com") {
```

Env variables without `NEXT_PUBLIC_` aren't available in the front end.

**Review by zomars — CHANGES_REQUESTED** [review]

File: pages/settings/profile.tsx [hunk 2]
```diff
@@ -124,8 +124,11 @@ function SettingsView(props: ComponentProps<typeof Settings> & { localeProp: str
     }).catch((e) => {
       console.error(`Error Removing user: ${props.user.id}, email: ${props.user.email} :`, e);
     });
-    // signout;
-    signOut({ callbackUrl: "/auth/logout" });
+    if (process.env.BASE_URL === "https://app.cal.com/") {
```
Comment (zomars) [inline pages/settings/profile.tsx:127]:
```suggestion
    if (process.env.NEXT_PUBLIC_BASE_URL === "https://app.cal.com") {
```

Env variables without `NEXT_PUBLIC_` aren't available in the front end.

**Review by zomars — CHANGES_REQUESTED** [review]

**Review by zomars — APPROVED** [review]

**Comment by PeerRich** [comment]
just made a test account and deleted it, it just logged me out and didn't redirect me to the cancellation survey

**Comment by alishaz-polymath** [comment]
> just made a test account and deleted it, it just logged me out and didn't redirect me to the cancellation survey

That's weird. There are only 2 possible reasons I can think of which I couldn't test locally:
1. The base-url is not exactly `https://app.cal.com`
2. For some reason, app.cal.com is not being considered as the same host name as https://cal.com and thus the callback url isn't working
[ref:The URL must be considered valid by the redirect callback handler. By default this means it must be an absolute URL at the same host name]

**Pull request merged**